src/client/windshaft.js

```diff
@@ -182,8 +185,11 @@ export default class Windshaft {
         return aggregation;
     }
 
-    _buildSelectClause(MRS) {
-        return MRS.columns.map(name => name.startsWith('_cdb_agg_') ? getBase(name) : name).concat(['the_geom', 'the_geom_webmercator']);
+    _buildSelectClause(MRS, dateFields = []) {
+        return MRS.columns.map(name => name.startsWith('_cdb_agg_') ? getBase(name) : name).map(
+            name => dateFields.includes(name) ? name + '::text' : name
```

Comment:
`::text` ? 

what is this?

Reply:
A hack, a workaround Maps API since Maps API skip SQL `date` properties if they are not casted like this. This will go away when Maps API improves timestamp support.